source/future.tex

```diff
@@ -1459,6 +1459,123 @@
 \end{note}
 \end{itemdescr}
 
+\rSec1[depr.tuple]{Tuple}
+
+Header \libheaderref{tuple}:
+
+\begin{codeblock}
+namespace std {
+  // \ref{tuple.helper}, tuple helper classes
```

Comment:
This reference should be to the deprecated \itemdecls which are now in this section.  Either add a [depr.tuple.helper] as the paper suggests and use that reference here or remove the reference.

I'd suggest just removing the reference since the section is so small.

Reply:
Fixed.